recipe/meta.yaml

```diff
@@ -1,18 +1,15 @@
-# Ported from Continuum Analytics: https://github.com/conda/conda-recipes/tree/master/r-base
-
-{% set name = "r-base" %}
-{% set native = 'm2w64-' if win else '' %}
-{% set posix= 'm2-' if win else '' %}
 {% set version = '3.3.2' %}
-{% set sha1 = "0e39e9c2d28fe6bab9c55ca23e08ba8727fd2fca" %}
+{% set native = 'm2w64-' if win else '' %}
 
 package:
-  name: {{ name|lower }}
+  name: r-base
```

Comment:
The name and the checksum are used only once so I removed the variables to keep the recipe clear,